src/transformers/training_args.py

```diff
@@ -240,6 +240,10 @@ class TrainingArguments:
         report_to (:obj:`List[str]`, `optional`, defaults to the list of integrations platforms installed):
             The list of integrations to report the results and logs to. Supported platforms are :obj:`"azure_ml"`,
             :obj:`"comet_ml"`, :obj:`"mlflow"`, :obj:`"tensorboard"` and :obj:`"wandb"`.
+        ddp_find_unused_parameters (:obj:`bool`, `optional`):
+            When using distributed training, the value of the flag :obj:`find_unused_parameters` passed to
+            :obj:`DistributedDataParallel`. Will defaut to :obj:`True` if gradient checkpointing is not used,
+            :obj:`False` otherwise.
```

Comment:
```suggestion
            :obj:`DistributedDataParallel`. Will default to :obj:`False` if model is ``PreTrainedModel``and  gradient checkpointing is used,
            :obj:`True` otherwise.
```

Reply:
Not sure about this one, this just adds the `PreTrainedModel` in the equation but says the same thing. I don't think we need to cover the case where the user is dealing with gradient checkpointing on their side (they probably have their custom training loop if that's the case) so we can leave that out, in which case it's just the same as I did without the negation. Maybe it's clearer this way, will push a commit.

Reply:
I see what I you mean, but how can this logic be described if it excludes a key part of `PreTrainedModel`?

If it's not a `PreTrainedModel` it is then `True` always, regardless of `gradient_checkpointing`

Reply:
There is no `gradient_checkpointing` attribute if the model is not a `PreTrainedModel`, it's a transformers thing.

Reply:
I see what you mean. Your latest incarnation reads well then. Thank you!